src/server/routes/garmin.ts

```diff
@@ -1,10 +1,55 @@
 import express from 'express';
+var bytes = require('bytes');
 
+const app = express();
```

Comment:
no needd to define a new express app. the global server app is defined in server/index.tsc and garminRouter is attached to it in routes.ts